dFusion/dFusion.rst

```diff
@@ -223,7 +222,8 @@ The anchor smart contract on ethereum will offer the following function:
 
 This function will update the rolling hash of pending orders, chaining all orders with a valid signature. 
 This function is callable by any party. 
-However, it is possible that “decentralized operators” accept orders from users, bundle them and then submit them all together in one function call. 
+It is possible that “decentralized operators” accept orders from users, bundle them and then submit them all together in one function call. 
```

Comment:
We lost our "however" here. Can we check that this sentence is still tied to the previous one?